source/fuzz/fuzzer_util.cpp

```diff
@@ -1342,7 +1342,47 @@ opt::Instruction* GetLastInsertBeforeInstruction(opt::IRContext* ir_context,
   return CanInsertOpcodeBeforeInstruction(opcode, &*it) ? &*it : nullptr;
 }
 
-}  // namespace fuzzerutil
+uint32_t MaybeGetLocalVariable(opt::IRContext* ir_context, uint32_t type_id,
+                               uint32_t function_id) {
+  uint32_t return_id = 0;
+  auto function = FindFunction(ir_context, function_id);
+
+  // All of the local variable declarations are located in the first block
+  auto block = function->begin();
```

Comment:
Make sure to have a reference here (or a constant reference which is better). Otherwise, we are creating a copy of an entry block.